app/assets/javascripts/discourse/app/components/emoji-picker.js

```diff
@@ -243,12 +243,111 @@ export default Component.extend({
     section && section.scrollIntoView();
   },
 
+  numEmojiPerRow(emojis) {
```

Comment:
I think it would be more efficient to calculate the number of emojis per row only once, right after the first render, and then that can be stored as a component property and used on the action. 

This will also help with the search results (reason why that isn't working now, I believe, is because search results markup doesn't match `.emojis-container .section-group`.)